### PR TITLE
[openfhe_julia] update to version v0.3.0

### PR DESCRIPTION
**Pull request by ArseniyKholod**

### CODE REVIEW

**Comment by ArseniyKholod** [comment]
@sloede, could you take a look?

**Review by sloede — APPROVED** [review]
LGTM, thanks!

File: O/openfhe_julia/build_tarballs.jl [hunk 1]
```diff
@@ -77,8 +77,8 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    BuildDependency(PackageSpec(;name="libjulia_jll", version=v"1.10.7")),
-    Dependency(PackageSpec(name="libcxxwrap_julia_jll", uuid="3eaa8342-bff7-56a5-9981-c04077f7cee7"); compat="0.11.2"),
+    BuildDependency(PackageSpec(;name="libjulia_jll", version=v"1.10.9")),
+    Dependency(PackageSpec(name="libcxxwrap_julia_jll", uuid="3eaa8342-bff7-56a5-9981-c04077f7cee7"); compat="0.12.2"),
```
Comment (giordano) [inline O/openfhe_julia/build_tarballs.jl:81]: I presume you're aware this may (or may note) require changes in your CxxWrap code?  Just checking!

Comment (sloede) [reply O/openfhe_julia/build_tarballs.jl:81]: Thanks for being vigilant though 🙏

Comment (ArseniyKholod) [reply O/openfhe_julia/build_tarballs.jl:81]: Thank you very much, yes we are aware!)

**Pull request merged**